tests/src/util/playgrounds/unique.dev.ts

```diff
@@ -69,11 +70,11 @@ class ArrangeGroup {
    * @returns array of newly created accounts
    * @example const [acc1, acc2, acc3] = await createAccounts([0n, 10n, 20n], donor); 
    */
-  creteAccounts = async (balances: bigint[], donor: IKeyringPair): Promise<IKeyringPair[]> => {
+  creteAccounts = async (balances: bigint[], donor: TSigner): Promise<TSigner[]> => {
     let nonce = await this.helper.chain.getNonce(donor.address);
```

Comment:
if `donor` type is `TSigner`, then we should use `helper.getSignerAddress(donor)` insted of donor.address

Reply:
Will return `IKeyringPair` here. I think we do not need TSigner in tests